src/main/java/org/eclipse/yasson/internal/serializer/DefaultSerializers.java

```diff
@@ -196,13 +194,13 @@ public boolean isKnownType(Class<?> clazz) {
 
         return knownContainerValueType || findValueSerializerProvider(clazz).isPresent();
     }
-
-    /**
-     * Singleton instance.
-     *
-     * @return instance
-     */
-    public static DefaultSerializers getInstance() {
-        return INSTANCE;
+    
+    private static boolean isClassAvailable(String className) {
+        try {
+            Class.forName(className);
+            return true;
+        } catch (ClassNotFoundException e) {
```

Comment:
Class.forName can also throw a `LinkageError` (superclass of `NoClassDefFoundError`) so lets include it in the catch block as well

Reply:
Makes sense, added it